packages/shared-ui/src/styles/index.css

```diff
@@ -261,7 +261,7 @@
 .cb-phone-input-field-input {
   flex: 4;
   border: 0px;
-  height: var(--cb-input-field-height);
+  height: calc(var(--cb-input-field-height) - 0.2rem);
```

Comment:
Is this a fix to the problem we were discussing? (that CSS frameworks like tailwind override styles for HTML elements like input, div, ...) => I thought we have to explicitly set some CSS for elements where Tailwind defines defaults that don't work with out stylings (the change above looks more like a quick fix to me though)

Reply:
Basically the design of our phone input is different then other input fields.
For other identifiers our container is of `input` type and has a height of `var(--cb-input-field-height)`. Also, inputs have borders.
For phone input our container is `div` since it has two components a selection dropdown button and an input container to add remaining phone number digits. In this scenario the div has borders and input borderless. Though the issue became prominent with tailwind's stylings, I think that this should also be how it should be implemented. Since the parent has borders the child should have a slightly smaller height then parent.